app/models/imported_item.rb

```diff
@@ -18,7 +18,7 @@ def import(params)
       contacts << Contact.new(
         test_and_trace_account_id: row[0],
         nhs_number: row[1],
-        is_vulnerable: (row[2] == 1 || row[2]&.downcase == 'true'),
+        is_vulnerable: (row[2] == 1 || row[2].to_s&.downcase == 'true'),
```

Comment:
good catch, thanks. We can remove `&` in this case.  It should be:
`is_vulnerable: (row[2] == 1 || row[2].to_s.downcase == 'true'),`

Reply:
makes sense 👍